services/frontend-v3/src/components/basicInfo/BasicInfoView.jsx

```diff
@@ -1,27 +1,31 @@
 import React from "react";
-import { FormattedMessage } from "react-intl";
+import { injectIntl, FormattedMessage } from "react-intl";
```

Comment:
You don't need injectIntl in this file, as you're not using intl in the props
```suggestion
import { FormattedMessage } from "react-intl";
```

Reply:
remove unneeded injectIntl import